ray-operator/config/crd/patches/cainjection_in_ray_rayjobs.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I fee we don't necessarily need these patches so we don't need to check them in? WDYT?